### DISCUSSION TITLE
help customising bookmark stylesheet - built-in style editor not working

### DISCUSSION THREAD

**Discussion by yotamguttman** [mbnuqw/sidebery#1501] [Q&A]
hello, I've been trying to manipulate the styling of the bookmarks tab using the Sidebery styles editor. 
I've managed to customise the tabs view entirely to my liking so the function works. it just does not affect anything other than the tabs tab.

in the screenshot below you can see that I've gone quite overboard with the styling, to test the effectivity of the feature, and you can see that the bookmarks tab still looks like the Firefox default. 

![Screenshot from 2024-03-04 10-09-44](https://github.com/mbnuqw/sidebery/assets/92977002/53b815b9-f716-43b0-87ac-e8fcbe86aacb)

is there a way to also customise the bookmarks tab (without entering the userChrome shenanigan)?
also, what's this section called 'Bookmarks' in the styles editor for if not for the bookmarks sidebar?

thanks for your help :)

**Comment by mbnuqw** [comment]
This is not Sidebery's bookmarks panel, it's default Firefox bookmarks sidebar, which can be customized only with userChrome.css. Sidebery's style editor can apply styles only on Sidebery content - sidebar or group-page. To turn on sidebery sidebar click on the drop-down menu ("Bookmarks" in screenshot) and select an option with sidebery's icon.

**Reply by yotamguttman** [reply]
thanks for your reply :)
so you mean this little panel that pops from the bottom right? because I still don't seem to be able to change it all that much. the only property I see affected is the 'node activated foreground'. I'm trying to make the line height more pronounced. I gave it a massive property but I don't see it changing. 

![Screenshot from 2024-03-04 11-50-53](https://github.com/mbnuqw/sidebery/assets/92977002/83360dcc-253f-4428-b5a5-a047d694cef8)

**Reply by mbnuqw** [reply]
Can't reproduce. Could you please open a new issue with more info: version of addon and your custom CSS for sidebery (as text)?
